multicluster/controllers/multicluster/common/helper.go

```diff
@@ -104,3 +104,7 @@ func HashLabelIdentity(l string) string {
 	hashValue := hex.EncodeToString(hash.Sum(nil))
 	return hashValue[:labelIdentityHashLength]
 }
+
+func IsMulticlusterService(service *corev1.Service) bool {
+	return service.Annotations[AntreaMCServiceAnnotation] == "true"
```

Comment:
Where `AntreaMCServiceAnnotation `is defined?

Reply:
I found it's already defined in `multicluster/controller/multicluster/common/types.go`, so just reuse it.